test/unit/deploy/basic_info_test.rb

```diff
@@ -15,7 +15,9 @@ class BasicInfoTest < ActiveSupport::TestCase
   end
 
   test 'custom release' do
-    System::Deploy.info.expects(:release).returns('4.5.1-CR1')
+    path = Rails.root.join('test', 'fixtures', 'deploy_info').expand_path
```

Comment:
Do we really need the expand_path? Not a problem just asking

Reply:
I don't know. Maybe not, but I just followed that I saw in:https://github.com/3scale/porta/blob/eaf4b508de85199ae0caaa268904880af5bdf445/lib/system/deploy.rb#L62